avocado/utils/network/interfaces.py

```diff
@@ -176,6 +176,19 @@ def is_link_up(self):
         """
         return self.is_admin_link_up() and self.is_operational_link_up()
 
+    def get_default_interface(self):
+        """Get the default interface.
+
+        :return: Default interface name.
+        """
+        output = run_command("ip route", self.host)
```

Comment:
You can use `ip route list default`

Reply:
Indeed, `ip -j route list default` is a good option here.

Reply:
ok, will do that.